pymarketstore/results.py

```diff
@@ -4,10 +4,13 @@
 
 
 def decode(packed):
+
     dt = np.dtype([
-        (colname, coltype)
+        (colname if isinstance(colname, str) else colname.encode("utf-8"),
```

Comment:
`isinstance(colname, unicode)` doesn't work?

Reply:
unicode is not part of python3 namespace, it is in python 2 though, str is present on both versions.

Reply:
ah makes sense. sounds good.